frontend/src/components/TheHeader.vue

```diff
@@ -35,9 +35,10 @@
       <template #menu>
         <div class="flex flex-row items-center justify-end h-full space-x-1">
           <!-- button class="border-none btn" @click="openView('/options')">Options</button -->
-          <button class="border-none btn" @click="router.push('/about')">About</button>
-          <button class="border-none btn">Source code
-          <i-ant-design-github-filled></i-ant-design-github-filled>
+          <button class="border-none btn" @click="$router.push('/about')">About</button>
+          <button class="border-none btn" @click="openSourceCode()">
```

Comment:
Hi
Can we add `target="_blank"` for external link?
It's better the link opens in new tab.

Reply:
We could open GitHub in an other window (or tab) …
But the user may also want to decide to open it  in a new window (or tab) themself.
I think the best is to avoid forcing the user to open GitHub in a new window (or tab).

Reply:
Hi @arad1990 Once on the GitHub page (the Rainbow Git repo), the user can click to go to the Rainbow website. So ne need to create a new window. As you can notice, the GitHub website does not open a new window when we click on the Rainbow website (external link)

Reply:
Hi @0uep I think It's possible that some users not go back to the project page if they click on external links and they open in the current tab not new tab.

Reply:
Hi @arad1990 Your feedback is interesting.

On my experience, I have never meet a user lost when a website reuses the same window while clicking on links. But I often see the opposite. For example, my mother in law is lost when a link opens a new tab (or new window), she does not understand why she cannot go backward when she is in the new tab (or new window).

I have one question for you. Please open a new tab and go to [http://github.com/teal-finance/rainbow](https://github.com/teal-finance/rainbow).
You can see the link **:link: [teal.finance/rainbow](https://teal.finance/rainbow)** on the right.
### Why the click on this link does not open a new tab (or window)?

Reply:
Hi @0uep.
I clicked on the [teal.finance/rainbow ](https://teal.finance/rainbow ) and it opens in new tab.
Also I don't want to waste your time and other for some subjects that it's not important!
I think It's OK without open in new tab and as you said before, it's better user decide about it.
Thanks.

Reply:
Oops Yes you are right, it opens a new tab/window. :open_mouth: 
I am using the Firefox Add-on https://github.com/refined-github/refined-github that avoids the opening of new tab/window while clicking on the GitHub page. 
Thank you for noticing it ;-)